.github/workflows/pr-custom-review.yml

```diff
@@ -0,0 +1,26 @@
+name: Check reviews
+
+on:
+  pull_request:
+    branches:
+      - master
+      - main
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - review_requested
+      - review_request_removed
+  pull_request_review:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: pr-custom-review
+        uses: paritytech/pr-custom-review@v1
+        with:
+          token: ${{ secrets.PRCR_TOKEN }}
+          locks-review-team: non-parity-devs
```

Comment:
I see that there's currently just Gav in that group, but its name seems utterly wrong here.

Reply:
Yes, exactly. I took the most suitable groups from existing. This PR should rise the conversation about groups composition and naming.